lib/github-release.rb

```diff
@@ -128,8 +133,15 @@ def git_config(item, default = nil)
 	end
 
 	def create_release(tag, prerelease)
-		print "Pushing to #{remote_name}...\n"
-		system("git push #{remote_name} tag #{tag}")
+		$pushed ||= false
```

Comment:
To be clear, this is my *first* attempt at programming in ruby.  I usually work in C or assembler.  That said, I did pick a global intentionally as the state needs to be preserved between calls to create_release.  I don't know if a @ variable would do that, but a global certainly does.